cmd/internal/cli/cache_clean_linux.go

```diff
@@ -74,15 +87,36 @@ var CacheCleanCmd = &cobra.Command{
 }
 
 func cacheCleanCmd() error {
-	// We create a handle to access the current image cache
+	if !cacheCleanForce {
+		ok, err := cacheCleanPrompt()
+		if err != nil {
+			return fmt.Errorf("could not prompt user: %v", err)
+		}
+		if !ok {
+			sylog.Infof("Cache cleanup canceled")
+			return nil
```

Comment:
A bit awkward moment. Do we need to return an error if deleting was canceled? I believe that yes, since the operating was not performed. Just to think.

Reply:
We can introduce `errCleanCancelled` here and move logging to the level up.

Reply:
I would better create a more generic err for cli operations. Something like `errOperationCancelled` So it can be used in other operations with user input.